ci/test/04_install.sh

```diff
@@ -42,7 +42,9 @@ if [ -z "$DANGER_RUN_CI_ON_HOST" ]; then
 
   if [ -n "${RESTART_CI_DOCKER_BEFORE_RUN}" ] ; then
     echo "Restart docker before run to stop and clear all containers started with --rm"
-    systemctl restart docker
+    podman container kill --all  # Similar to "systemctl restart docker"
```

Comment:
I'm not sure this works? Even though the CLIs have a compatible syntax, I think they operate on their own set of resources. I'm pretty sure `podman ps` right before this line won't show anything, so I think no containers are killed after this line? At least, that's what I get when I try locally spinning up a container with `docker run` and then try to show/kill it with `podman ps` or `podman container kill --all`.

Reply:
Okay, I can't comment on that. Since there is no mention of `podman` anywhere else in the codebase/documentation, I think this may make maintaining the CI more opaque, but perhaps that's an okay tradeoff.